import_export/admin.py

```diff
@@ -489,6 +489,13 @@ def import_action(self, request, *args, **kwargs):
                     dataset = input_format.create_dataset(data)
                 except Exception as e:
                     self.add_data_read_fail_error_to_form(import_form, e)
+                else:
+                    if len(dataset) == 0:
+                        import_form.add_error(
+                            "import_file",
+                            "No valid data to import. Ensure your file "
```

Comment:
Please can you wrap this message in the translation [alias](https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#standard-translation) so that we can define error messages in other languages.

Reply:
Done